From c35065bc3959785db3b88648b102478df34e511c Mon Sep 17 00:00:00 2001
From: Pratik Shandilya <pratikshandilya.12@gmail.com>
Date: Sun, 2 Jun 2019 03:56:46 +0000
Subject: [PATCH 28/36] fix ls navbar back button after launching camera.

fixes https://gitlab.com/LineageOS/issues/android/issues/767
Steps to reproduce:
1.Turn screen on (in lockscreen)
2.Double tap power button or use the lockscreen shortcut to open camera
3.Tap back button
4.Swipe up to show pattern/pin view and check if back button is showing

Change-Id: I475739401fd93ce14b47da2c7d52e6ff876c51ed
---
 .../SystemUI/src/com/android/keyguard/KeyguardHostView.java   | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/packages/SystemUI/src/com/android/keyguard/KeyguardHostView.java b/packages/SystemUI/src/com/android/keyguard/KeyguardHostView.java
index 375dd9e49e8..509df301b80 100644
--- a/packages/SystemUI/src/com/android/keyguard/KeyguardHostView.java
+++ b/packages/SystemUI/src/com/android/keyguard/KeyguardHostView.java
@@ -423,10 +423,6 @@ public class KeyguardHostView extends FrameLayout implements SecurityCallback {
     @Override
     public void dispatchSystemUiVisibilityChanged(int visibility) {
         super.dispatchSystemUiVisibilityChanged(visibility);
-
-        if (!(mContext instanceof Activity)) {
-            setSystemUiVisibility(STATUS_BAR_DISABLE_BACK);
-        }
     }
 
     /**
-- 
2.17.1

